README.md

```diff
@@ -0,0 +1,186 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Blue-ray Pay <a name="about-project"></a>
+
+**Blue-ray Pay** is a classic example of a payment tracking application, built using the Ruby on Rails Frawework.
+The application is a fully functional app that allows users to sign-up/log-in, create different categories & create different transactions that belong to one or many categories.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- Ruby
+- Ruby on Rails
+- Postgresql (Database)
+- Haml
+- Tailwind
+- RSpec & Capybara for testing
+
+### Key Features <a name="key-features"></a>
+
+- Athentication (using devise)
+- Image upload, validation & processing (using "shrine" & "image_processing" gems)
+- Responsive design with tailwind
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Coming soon when the project is finished
+
```

Comment:
- [ ] As is a [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md)for this project, "`Technical requirements: The project should be deployed and accessible online.`" Please deploy your application using either `Heroku, Netlify or GitHub Pages`.
Please make sure your project is deployed properly, the home and detail page should work as expected. A link to the deployed project is present in the README. I understand from your video presentation that you have some challenges to deploy your app due to payments, the above mentioned platform have free tiers, please kindly use one of them. Checkout this [link ](https://devcenter.heroku.com/articles/getting-started-with-rails7)on how to deploy your app with Heroku.

Reply:
@Mihndim2020  Unfortunately, Heroku doesn't allow me to sign up even & after asking some of my friends, they told me that Heroku is no longer free.
Netlify & GitHub pages can't deploy rails applications.
So what now? :joy:

Reply:
Needs attention!